### PR TITLE
i18n(es): Translated `vue.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `vue.mdx` to spanish 😄

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/vue.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import Video from '~/components/Video.astro';
+
+
+Esta **[integración de Astro][astro-integration]** permite el renderizado del lado del servidor y la hidratación del lado del cliente para tus componentes [Vue 3](https://vuejs.org/).
+
+## Instalación
+
+Hay dos maneras de agregar integraciones a tu proyecto. ¡Probemos la opción más conveniente primero!
+
+### `astro add` command
```
Comment (vickvasquez) [inline src/content/docs/es/guides/integrations-guide/vue.mdx:20]:
```suggestion
### Comando `astro add`
```

Comment (vickvasquez) [inline src/content/docs/es/guides/integrations-guide/vue.mdx:20]:
```suggestion
### Comando `astro add`
```

File: src/content/docs/es/guides/integrations-guide/vue.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+Astro inlcuye una herramienta CLI para agregar integraciones de primera parte: `astro add`. Este comando:
+
+1.  (Opcinalmente) Instalará todas las dependencias y dependencias de pares necesarias
```
Comment (vickvasquez) [inline src/content/docs/es/guides/integrations-guide/vue.mdx:24]:
```suggestion
1.  (Opcionalmente) Instala todas las dependencias y dependencias de pares necesarias
```

File: src/content/docs/es/guides/integrations-guide/vue.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+2.  (También opcionalmente) Actualizar tu archivo `astro.config.*` para aplicar esta integración
```
Comment (vickvasquez) [inline src/content/docs/es/guides/integrations-guide/vue.mdx:25]:
```suggestion
2.  (También opcionalmente) Actualiza tu archivo `astro.config.*` para aplicar esta integración
```

File: src/content/docs/es/guides/integrations-guide/vue.mdx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+export default defineConfig({
+  // ...
+  integrations: [vue()],
+});
+```
+
+## Empezando
+
+Para usar tu primer componente React en Astro, dirígete a nuestra [documentación de framework UI][astro-ui-frameworks]. Explorarás:
```
Comment (vickvasquez) [inline src/content/docs/es/guides/integrations-guide/vue.mdx:70]:
```suggestion
Para usar tu primer componente Vue en Astro, dirígete a nuestra [documentación de framework UI][astro-ui-frameworks]. Explorarás:
```

File: src/content/docs/es/guides/integrations-guide/vue.mdx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+```js
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  integrations: [
+    vue({
+      jsx: {
+        // trata cualquiere etiqueta que comience con ion- como elementos personalizados
```
Comment (vickvasquez) [inline src/content/docs/es/guides/integrations-guide/vue.mdx:173]:
```suggestion
        // trata cualquier etiqueta que comience con ion- como elementos personalizados
```

**Review by vickvasquez — COMMENTED** [review]
Te dejo uns comentarios.

**Comment by dreyfus92** [comment]
Muchas gracias por el review @vickvasquez 😄

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @dreyfus92 and @vickvasquez for the review 🙌

**Pull request merged**